README.md

```diff
@@ -20,7 +24,7 @@ This is not an officially supported Google product.
 
 ## Requirements
 
- * Recommended: Configure at least one target graphics API.  See below.
+ * Recommended: Configure at least one target graphics API. See below.
```

Comment:
Two spaces after a period is correct.

Reply:
Correctly wasting spaces. Didn't you know, spaces are like helium and there is a finite number of them on Earth. You can't just waste them.